src/backend/database/History.model.js

```diff
@@ -0,0 +1,17 @@
+const { DataTypes } = require("sequelize");
+const sequelize = require("./connection.js");
+
+const History = sequelize.define("History", {
+  id: {
+    type: DataTypes.STRING,
```

Comment:
change the `id` type to `UUID` v4. [see this example](https://sequelize.org/docs/v7/models/data-types/#built-in-default-values-for-uuid) for more detail

Reply:
alright sir